app/assets/images/icons/icon-facebook.svg

```diff
@@ -0,0 +1,841 @@
+
+
+
+
+
+
+<!DOCTYPE html>
```

Comment:
This file doesn't seem right @105ron, it looks like the source code for the home page of the site?